test/versioning/hex.spec.ts

```diff
@@ -1,115 +1,104 @@
-import { api as semver } from '../../lib/versioning/hex';
+import { api } from '../../lib/versioning/hex';
```

Comment:
Why this change?

Reply:
I've renamed it to `hexScheme` just to not confuse with actual `semver` scheme